plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -1142,14 +1142,20 @@ private static ObjectWriteFunction longTimestampWriteFunction()
     @Override
     public void setColumnComment(ConnectorSession session, JdbcTableHandle handle, JdbcColumnHandle column, Optional<String> comment)
     {
+        // PostgreSQL doesn't support prepared statement for COMMENT statement
         String sql = format(
                 "COMMENT ON COLUMN %s.%s IS %s",
                 quoted(handle.asPlainTable().getRemoteTableName()),
                 quoted(column.getColumnName()),
-                comment.isPresent() ? format("'%s'", comment.get()) : "NULL");
+                comment.isPresent() ? format("'%s'", escape(comment.get())) : "NULL");
```

Comment:
It's better to have addition of surrounding `'` and escaping intra-value `'` in a single place.

```
comment.map(PostgreSqlClient::varcharLiteral).orElse("NULL")
```

```java
private static String varcharLiteral(String value) {
{
   return "'"  + value.replace("'", "''") + "'";
}
```

Reply:
BTW this is SQL-standard behavior, it doesn't need to be implemented in this class, i.e. every conformant DB should behave like this, so it can be provided by the base-jdbc